src/main/java/herrekt/Storage.java

```diff
@@ -0,0 +1,81 @@
+package herrekt;
+
+import herrekt.taskmanager.Deadline;
+import herrekt.taskmanager.Event;
+import herrekt.taskmanager.Task;
+import herrekt.taskmanager.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.util.stream.Collectors.toList;
+
+public class Storage {
+
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public List<Task> load() throws FileNotFoundException {
+        List<String> stringList = loadSaveAsStringList(filePath);
+        return convertStringListToTaskList(stringList);
+    }
+
+    public void save(TaskList tasks) {
+        try {
+            if (tasks.getSize() == 0) {
+                FileWriting.writeToFile("save.txt", "");
+            } else {
+                StringBuilder toSaveToFile = tasks.convertTaskListToSaveFormat();
+                FileWriting.writeToFile("save.txt", toSaveToFile.toString());
+
+            }
+        } catch (IOException e) {
+            System.out.println("ERROR! FILE NOT FOUND!");
+        }
+    }
+
+    public List<String> loadSaveAsStringList(String filePath) throws FileNotFoundException {
+        File f = new File(filePath);
+        List<String> taskInStringList = new ArrayList<>();
+        Scanner sc = new Scanner(f);
+        while (sc.hasNext()) {
+            taskInStringList.add(sc.nextLine());
+        }
+        return taskInStringList;
```

Comment:
This is better than the `return` statement that does a method call. It improves the readability of your code.